src/lib.rs

```diff
@@ -74,23 +75,26 @@
 //!     .unwrap();
 //! let pair2 = (
 //!     vec![1, 2, 3],
-//!     Felt252Wrapper::from_hex_be("0x66342762FDD54D033c195fec3ce2568b62052e").unwrap(),
+//!     Felt::from_hex("0x66342762FDD54D033c195fec3ce2568b62052e").unwrap(),
 //! );
 //! bonsai_storage
 //!     .insert(&BitVec::from_vec(pair2.0.clone()), &pair2.1)
 //!     .unwrap();
 //! bonsai_storage.commit(id_builder.new_id()).unwrap();
 //! ```
+#![cfg_attr(not(feature = "std"), no_std)]
+extern crate alloc;
```

Comment:
this is only used in `no_std`. It should be behind `#[cfg(not(feature = "std"))]`